src/components/VColorPaletteMaterial.vue

```diff
@@ -288,5 +294,4 @@ export default {
     }
   }
 }
-
```

Comment:
Do we need this?

 **&-base** {
      width: 40px;
    }
    **&-light** {
      color: black;
    }